src/ORM/Connect/DBSchemaManager.php

```diff
@@ -398,7 +398,7 @@ public function requireTable(
             foreach ($fieldSchema as $fieldName => $fieldSpec) {
                 //Is this an array field?
                 $arrayValue = '';
-                if (strpos($fieldSpec ?? '', '[') !== false) {
+                if (strpos($fieldSpec ?? '', '[') !== false && strtolower(substr($fieldSpec, 0, 4)) !== 'enum') {
```

Comment:
I'm on the fence about this.... because yes, this is the way to fix this. But also, `DBEnum` does specifically use `arrayValue` in its `requireField()` method.

Unfortunately I can't find any information about what this _does_ so.... it's unclear what ramifications this could have. Did you look into that at all? If so can you please help ease my mind about this?

Reply:
Yeah good spot. In practice it's probably never used on Enum, though best to be sure. Have redone solution to make it robust